src/Morph/Web/MorphSelectOverrideDialog.qml

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2013-2016 Canonical Ltd.
+ * Copyright 2016-2020 UBports Fondation
```

Comment:
Fo**u**ndation

If this code was not copied from another file with a similar copyright, there is no need to have the 2013-2020 copyright chain. Instead, just `2020 UBports Foundation` is enough.